tests/qir/test_pytket_qir_20-QIRProfile.BASE.ll

```diff
@@ -0,0 +1,71 @@
+; ModuleID = 'test_pytket_qir_20'
+source_filename = "test_pytket_qir_20"
+
+%Qubit = type opaque
+%Result = type opaque
+
+@0 = internal constant [5 x i8] c"c[0]\00"
```

Comment:
We need to record the results individually, there might be device specific options to do this is a better way. I think this is the best way to do it compatible to the base profile.